introduction.tex

```diff
@@ -1,15 +1,31 @@
 \section{Introduction} \label{sec:intro}
 
-The current situation is  that, only the Science Pipelines is released as follows:
+This document collects the needs in therms of releses from different stakehoders, section \ref{sec:reqs}.
+These needs are presented as requirements, that shall not be confused with project requirements, like for example the \gls{DMS} requirement specification, documented in \citeds{LSE-61}.
+Section \ref{sec:process} provides an overview of the release process.
+
+Appedix \ref{sec:defs} provides definitions of the main objects involved in the release activities. 
+Those definitions are given in order to provide a common understanding.
+In the same line, appendix \ref{sec:swptree} gives an overview of the software products part of the \gls{DM}, that are involved in this process.
+
+
+\newpage
+\section{Science Pipelines Releases}\label{sec:sci}
+
+The current situation is that only the Science Pipelines product is released regularly.
+The process in place provides time based builds and releases:
 
 \begin{itemize}
-\item Stable releases every 6 months
-\item Weekly builds
 \item Nightly builds
+\item Weekly builds
+\item Official releases every 6 months
 \end{itemize}
 
-The current release process requires few weeks to make an official release available.
+The time needed to consolidate an official release from a weekly build is considerable.
+Usually 2 or 3 weeks are sufficient but in some cases it may take more than a month. 
+Technical details and problems are given in \citeds{DMTN-106}.
 
-For this reasons weekly builds are largely used, since they provide the most up to date and stable software.
+Due to the time issue, when a release is available to the users, it is already too old.
```

Comment:
Consequently,  when a release is available to the users, it is already too old.

(remove the paragraph break)